### ISSUE TITLE
[Bug] App shows dark header with Light Theme

### ISSUE THREAD

**Issue by Dav2070** [jenius-apps/nightingale-rest-api-client#196]
As you can see on the screenshot below, on Light Mode the entire header is dark.

![image](https://user-images.githubusercontent.com/16546287/120838914-a591d280-c568-11eb-8c08-eb9d2452a3b5.png)

I'm using Windows 10 21H1 and Nightingale v4.4.0.

**Comment by dpaulino** [comment]
if you restart, does the issue get fixed?

**Comment by Dav2070** [comment]
No, even rebooting doesn't fix it. It only works correctly when manually switching to Light Theme in the Settings of the app. But when I use the System setting, I can't fix it at all.
